SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -33,11 +33,12 @@ define_repo_path() {
 
 # Function for printing the help page
 print_help() {
-    _log "Usage: makenv --repo/-r REPOSITORY [--repo_type TYPE] [--accpy ACCPY_VERSION] [--help/-h]"
+    _log "Usage: makenv --repo/-r REPOSITORY [--repo_type TYPE] [--accpy ACCPY_VERSION] [--conda] [--help/-h]"
```

Comment:
To tidy this up now, we could have an argument called `--builder`, and that can be `venv`, `accpy` or `conda`, venv can be the default.
Then, for the version, we could have `--builder_version`, to be able to set it for `accpy`. What do you think?

Reply:
Yup, that's exactly the change that I mention in the other PR. But that involves making a considerable amount of changes. That's why I think would make more sense to implement in a separate PR.

Reply:
Why so many changes? I would assume this only requires changing the way you call the `makenv.sh` script, which only happens in one place in the code of the SwanCustomEnvs extension, right?